azure-pipelines-official.yml

```diff
@@ -54,31 +47,13 @@ stages:
   - job: OfficialBuild
     displayName: Official Build
     timeoutInMinutes: 360
-    # Conditionally set build pool so we can share this YAML when building with different pipeline (devdiv vs dnceng)
+    # Conditionally set build pool so we can share this YAML when building with different pipeline
```

Comment:
We probably can remove all the checks for dnceng pipeline from this file too, or if we want to be sure, have a single, separate task at the beginning that fails the build if not running in dnceng.

Reply:
Removed!